TaskHouseUnitTests/EmpolyerControllerUnitTests.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using Xunit;
+using TaskHouseApi.Controllers;
+using TaskHouseApi.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using TaskHouseApi.Model;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace TaskHouseUnitTests
+{
+    public class EmpolyerControllerUnitTests
+    {
+        EmployerrsController controller;
+        IEmployerRepository repo;
+
+        public EmpolyerControllerUnitTests()
+        {
+            repo = new  FakeEmppolyerRepository();
+            controller = new EmployerrsController(repo);
+        }
+        
+        /// Test Get all
+        [Fact]
+        public async void EmpolyerController_Get_ReturnsAllElementsInRepo_WhenGivenNoParameters()
+        {
+            //Arrange and act
+            IEnumerable<Employer> result = await controller.Get();
+
+            //Asserts
+            Assert.Equal(3, result.Count());
+
+        }
+
+        ///Test Get with valid Id as parameter
+        [Fact]
+        public async void EmpolyerController_Get_ReturnsObjectReponseWithCorrectEmpolyer_WhenGivenValidId() 
+        { 
+            //arrange
+            int empolyerId = 2;
+
+            //Act
+            var result = await controller.Get(empolyerId);
+            var resultAsObject = await controller.Get(empolyerId) as ObjectResult;
+            var resultObject = resultAsObject.Value as Employer;
+
+            //Assert
+            var assertResult = Assert.IsType<ObjectResult>(result);
```

Comment:
Remove the varible declaration, no need to do anytign but assert the type (my mistake originally though)